### PR TITLE
Better support for Kibana auth.

### LINKED ISSUE #129: Add authentication config for Kibana
We can integrate this into the setup script w/ Ansible. Already have some ideas to make this easy.

### PR DESCRIPTION
**Pull request by jeffgeiger**
*  Adds `enable_kibana_auth` and `disable_kibana_auth`
 *  Adds `kibanapw` to edit users and passwords.
Fixes #129

### CODE REVIEW

**Comment by dcode** [comment]
@jeffgeiger Can we add it to be enabled by default? Maybe add `analyst` and a random password that is saved to the filesystem? This is similar to how openstack does it. Alternatively, we could use the same username to the admin user `getent passwd 1000 | cut -d: -f1`, defaulting to `admin` if they only set root password. Alternatively, we could allow passthrough to pam (and narrow to specific group membership) and allow either pam or htpasswd to satisfy the auth. This would require an RPM for the pam auth module.

I'm just spitballin'. It'd be nice to have auth by default.

**Comment by jeffgeiger** [comment]
I looked at the PAM route. Beyond not having a second password mgmt mechanism, it seemed to be a lot more work, and fraught with more peril than necessary. That said, I can add a routine that generates a random password and also writes it to something like a new sensor README written to the admin user's home directory.  

I skipped the "on by default", pending discussion, because it makes documentation updates a blocker for 2.1 release.  (Though I plan to drag through those with a backhoe this weekend anyway.)

**Comment by jeffgeiger** [comment]
I'll add this to the build to generate the default password and add it's availability to documentation:
https://packagecloud.io/rocknsm/2/packages/el/7/xkcdpass-1.11.1-1.noarch.rpm

**Comment by jeffgeiger** [comment]
@dcode - Should be ready for review now. 👀

File: playbooks/deploy-rock.yml [hunk 1]
```diff
@@ -1136,6 +1136,17 @@
       group: root
     when: with_kibana
 
+  - name: Set initial Kibana credentials
+    shell: >
+      kibuser=$(getent passwd 1000 | awk -F: '{print $1}')
```
Comment (dcode) [inline playbooks/deploy-rock.yml]: Does the Initial Setup script force a user 1000 creation? If not, maybe this (need to validate; not sure if error propagates)
```
kibuser=$((getent passwd 1000 | awk -F: '{print $1}') || echo 'root')
```

Comment (jeffgeiger) [inline playbooks/deploy-rock.yml]: Any user created by initial setup will be 1000, unless someone has really monkeyed with the ISO creation.  But in our build, the first user created will always be 1000.

**Pull request merged**